cefi/handler/capitalcom.py

```diff
@@ -49,19 +49,8 @@ def __init__(
         """
         super().__init__(**kwargs)
         try:
-            if self.testmode:
-                self.client = DemoClient(
-                    log=self.user_id,
-                    pas=self.password,
-                    api_key=self.api_key,
-                )
-            else:
-                self.client = Client(
-                    log=self.user_id,
-                    pas=self.password,
-                    api_key=self.api_key,
-                )
 
+            self._build_client()
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (docstrings):** Please update the docstring for function: `__init__`

Reason for update: The client initialization logic has been moved to a separate method.

Suggested new docstring:
```
Initializes the handler. Client initialization is now handled by _build_client method.
```